### PR TITLE
9 get comments by article ID

### PR DESCRIPTION
**Pull request by AlistairHopson**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -232,3 +232,48 @@ describe("GET /api/articles", () => {
...
+            author: "butter_bridge",
+            votes: 1,
+          })
+        );
+      });
+  });
+  test("If an article has no comments, returns 404 error", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(404)
```
Comment (douglashellowell) [inline __tests__/app.test.js:274]: This article exists but it just so happens that it has no comments!

A `404` would be misleading and might cause users to think the article didn't exist

This should be a `200` returning an empty array!

Comment (douglashellowell) [reply __tests__/app.test.js:274]: `GET /api/artices/99999/comments` is another story - make sure you include a test for this as well!

**Review by douglashellowell — CHANGES_REQUESTED** [review]
Some tweaks needed to your tests here please!

please put in another `/nchelp pr` when ammended 👍

**Comment by AlistairHopson** [comment]
Thanks - I've added further tests here

**Comment by P-Copley** [comment]
Looking good @AlistairHopson 

Go ahead and merge when you're ready :+1:

**Pull request merged**